cmd/slinky/main.go

```diff
@@ -343,3 +357,57 @@ func overwriteMarketMapEndpoint(cfg config.OracleConfig, overwrite string) (conf
 
 	return cfg, fmt.Errorf("no market-map provider found in config")
 }
+
+// filterToOnlyCMCMarkets is a helper function that filters out all markets that are not from CoinMarketCap. It
+// mutates the marketmap to only include CoinMarketCap markets. Notably the CMC ID will be pricing in the base
+// asset.
+func filterToOnlyCMCMarkets(marketmap mmtypes.MarketMap) mmtypes.MarketMap {
+	res := mmtypes.MarketMap{
+		Markets: make(map[string]mmtypes.Market),
+	}
+
+	// Filter out all markets that are not from CoinMarketCap.
+	for _, market := range marketmap.Markets {
+		var meta metaDataJSON
+		if err := json.Unmarshal([]byte(market.Ticker.Metadata_JSON), &meta); err != nil {
+			continue
+		}
+
+		var id string
+		for _, aggregateID := range meta.AggregateIDs {
+			if aggregateID.Venue == "coinmarketcap" {
+				id = aggregateID.ID
+				break
+			}
+		}
+
+		if len(id) == 0 {
+			continue
+		}
+
+		resTicker := market.Ticker
+		resTicker.MinProviderCount = 1
+
+		providers := []mmtypes.ProviderConfig{
+			{
+				Name:           coinmarketcap.Name,
+				OffChainTicker: id,
+			},
+		}
+
+		res.Markets[resTicker.CurrencyPair.String()] = mmtypes.Market{
+			Ticker:          resTicker,
+			ProviderConfigs: providers,
+		}
+	}
+
+	return res
+}
+
+type metaDataJSON struct {
+	AggregateIDs []aggregateIDsJSON `json:"aggregate_ids"`
+}
+type aggregateIDsJSON struct {
+	Venue string `json:"venue"`
+	ID    string `json:"ID"`
+}
```

Comment:
Any reason we shouldn't be using https://github.com/skip-mev/slinky/commit/df1036cf5ead09a2e20f4165857fe14b4314b197 here?

Reply:
Did not realize that was in the code base.